app/views/books/show.html.erb

```diff
@@ -2,6 +2,7 @@
 
 <h1>Title: <%= @book.title %></h1>
 <p>Species: <%= @book.species %></p>
+<p>category: <%= @book.category.name %></p>
```

Comment:
I'd suggest extracting `category.name` to instance method in Book model:
```
class Book < ActiveRecord::Base
  def category_name
    self.category.name
  end
end
```
This way your view is less coupled with models. Now you expect that you have book that responds to method category which returns object responding to name method.